docs/install/osx_setup.md

```diff
@@ -145,12 +155,16 @@ We have installed MXNet core library. Next, we will install MXNet interface pack
 To install the MXNet Python binding navigate to the root of the MXNet folder then run the following:
 
 ```bash
-$ cd python
-$ pip install -e .
+virtualenv -p`which python3` mxnet_py3
+source mxnet_py3/bin/activate
+pip install -e python
 ```
-
+First we create a [virtualenv](https://docs.python-guide.org/dev/virtualenvs/#lower-level-virtualenv) to isolate this installation from our global environment.
```

Comment:
This should not be required for installing mxnet.

Reply:
This is a good practice not to polute global environments.

Reply:
Removed